src/App.tsx

```diff
@@ -10,22 +10,34 @@ import { MyTimer } from './components/Timer';
 import { usePrevious } from './hooks/usePrevious';
 import MyButton from './components/MyButton';
 import { useMyCallback } from './hooks/useMyCallback';
+import UserData from './mock/userdata';
+import { User } from './models/user';
+import MyList from './components/MyList/MyList';
 
 function App() {
   const [showTime, setShowTime] = useState(true);
   const previousShowTime = usePrevious(showTime);
   const btnContent = useMemo(() => <>show time</>, []);
+  const [userData, setUserData] = useState<User[]>([]);
 
   const changeShowTime = useMyCallback(() => {
     setShowTime((prevValue) => !prevValue);
   }, []);
 
+  useEffect(() => {
+    // Fake fetching
+    setTimeout(() => {
+      setUserData(UserData);
+    }, 800);
```

Comment:
If your Effect fetches something, the cleanup function should either [abort the fetch](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) or ignore its result

- https://react.dev/learn/synchronizing-with-effects#fetching-data

Reply:
Good to know this one! added